resources/home/dnanexus/utils/util_functions.py

```
@@ -1,7 +1,7 @@
 """General utility functions"""
 
 import os
-from typing import Tuple, Union
+from typing import List, Tuple, Union
```

Ruff (0.8.0)

`typing.List` is deprecated, use `list` instead

(UP035)